bookmarks-list.html

```diff
@@ -0,0 +1,111 @@
+          title="Bookmark this Quiz Card"
+        ></a>
+        <details class="quiz-card__details">
+          <summary class="quiz-card__button">
+            <h3 class="quiz-card__headline">What is CSS?</h3>
+          </summary>
+          <p class="quiz-card__answer">
+            CSS stands for Cascading Style Sheets. It is a style sheet language
+            which is used to describe the look and formatting of a document
+            written in markup language. It provides an additional feature to
+            HTML. It is generally used with HTML to change the style of web
+            pages and user interfaces. It can also be used with any kind of XML
+            documents including plain XML, SVG and XUL. CSS is used along with
+            HTML and JavaScript in most websites to create user interfaces for
+            web applications and user interfaces for many mobile applications.
+          </p>
+        </details>
+        <div class="badge-group margin-top padding-top border-top">
+          <span class="badge-group__badge">Bookmarked</span>
+          <span class="badge-group__badge">CSS</span>
+          <span class="badge-group__badge">Beginner</span>
+          <span class="badge-group__badge">Intro</span>
+          <span class="badge-group__badge">Basics</span>
+        </div>
+      </section>
+      <section class="quiz-card">
+        <a
+          href="#"
+          class="quiz-card__bookmark quiz-card__bookmark--active icon-bookmark1"
+          title="Bookmark this Quiz Card"
+        ></a>
+        <details class="quiz-card__details">
+          <summary class="quiz-card__button">
+            <h3 class="quiz-card__headline">
+              Why background and color are the separate properties if they
+              should always be set together?
+            </h3>
+          </summary>
+          <div class="quiz-card__answer">
+            <p>There are two reasons behind this:</p>
+            <ul>
+              <li>
+                It enhances the legibility of style sheets. The background
+                property is a complex property in CSS, and if it is combined
+                with color, the complexity will further increase.
+              </li>
+              <li>
+                Color is an inherited property while the background is not. So
+                this can make confusion further.
+              </li>
+            </ul>
+          </div>
+        </details>
+        <div class="badge-group margin-top padding-top border-top">
+          <span class="badge-group__badge">Bookmarked</span>
+          <span class="badge-group__badge">background</span>
+          <span class="badge-group__badge">CSS</span>
+          <span class="badge-group__badge">Intro</span>
+          <span class="badge-group__badge">Basics</span>
+        </div>
+      </section>
+    </main>
+    <footer class="app-grid__footer">
+      <nav class="app-navigation">
```

Comment:
Simply `navigation` as class name would be enough, but it's up to you.
```suggestion
      <nav class="navigation">
```